package.json

```diff
@@ -10,6 +10,7 @@
     "license": "Apache-2.0",
     "author": "Cloudify Platform Ltd. <cosmo-admin@cloudify.co>",
     "files": [
+        "backend/*.{js,md}",
```

Comment:
Just question: in Stage and Composer, we would be able to import new code using eg.:
`const { runMigration } = require('cloudify-ui-common/backend');`
?

Reply:
Yes, exactly like that